static/js/popovers.js

```diff
@@ -497,12 +499,12 @@ export function toggle_actions_popover(element, id) {
         const should_display_uncollapse =
             !message.locally_echoed && !message.is_me_message && message.collapsed;
 
+        const should_display_reportmsg = !message_deleted && !message.sent_by_me && not_spectator;
```

Comment:
Hmm, why should messages with "(deleted)" not be possible to report? I would think it might be something one would want to do -- those messages usually still have edit history, for example.

Reply:
DONE - thanks for catching this.